### PR TITLE
Added Deploy Information

### PR DESCRIPTION
**Pull request by boldtrn**
As discussed in #593 I added information for deployment. Not sure if we have to add more information?

### CODE REVIEW

File: docs/core/deploy.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Important notes:
+
+ * jsonp support needs to be enabled in the config.properties
+ * none-hierarchical graphs should be limited to a certain distance otherwise you'll require lots of RAM per request! See [#104](https://github.com/graphhopper/graphhopper/issues/104)
+ * if you have strange speed problems which could be related to low memory you can try to [entire disable swap](http://askubuntu.com/questions/103915/how-do-i-configure-swappiness). Or just try it out via `sudo swapoff -a`. Swapping out is very harmful to Java programs especially when the GC kicks in.
+
+### API Tokens
+
+By default, graphhopper uses [Omniscale](http://omniscale.com/) as layer service. 
+Either you get a plan there or you have to remove Omniscale from the [JavaScript file](https://github.com/graphhopper/graphhopper/blob/master/web/src/main/webapp/js/main.js).
```
Comment (karussell) [inline docs/core/deploy.md]: switching to a different default is sufficient IMO

File: docs/core/deploy.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ 
+For simplicity you could just start jetty from maven and schedule it as background job: 
+`export GH_FOREGROUND=false && export JETTY_PORT=11111 && ./graphhopper.sh web europe_germany_berlin.pbf`. 
+Then the service will be accessible on port 11111.
+
+For production usage you can install the latest jetty (8 or 9) as a service but we prefer to have it bundled as a 
+simple jar. Tomcat should work too. To create a war file do `mvn clean install war:war` and copy it from the target/ 
+folder to your jetty installation. Then copy web/config.properties also there and change this properties 
+file to point to the required graphhopper folder. Increase the Xmx/Xms values of your jetty server e.g. 
+for world wide coverage with a hierarchical graph I do the following in bin/jetty.sh
```
Comment (karussell) [inline docs/core/deploy.md]: remove 'I'

File: docs/web/quickstart.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ The following steps are simpler and only need the JRE, a jar file and an OSM fil
  3. Start GraphHopper Maps via: `java -jar *.jar jetty.resourcebase=webapp config=config-example.properties osmreader.osm=berlin-latest.osm.pbf`
  4. After you see 'Started server at HTTP 8989' go to [http://localhost:8989/](http://localhost:8989/) and you should see a map of Berlin. You should be able to click on the map and a route appears.
 
+If you want to setup graphhopper on your own server, we have a separate information page [Deploy](./../core/deploy.md).
```
Comment (karussell) [inline docs/web/quickstart.md]: Hmmh, the problem here is that no git, bash etc is necessary but for the linked deploy.md it is. I would just say something like "To see how GraphHopper is configured for production usage, see the separate site" as the setup of graphhopper on own server is exactly what is done here :)

File: docs/core/deploy.md (hunk 1 above)
Comment (boldtrn) [inline docs/core/deploy.md]: You can still select it from the dropdown (top-right). If you select it you get tiles that tell the user that the API key is wrong, but no map is drawn.

File: docs/web/quickstart.md (hunk 3 above)
Comment (boldtrn) [inline docs/web/quickstart.md]: Are there actually these dependencies? In both cases they have to execute something on the command line. But they don't have to install anything?

Comment (karussell) [inline docs/web/quickstart.md]: for the graphhopper.sh script you need not only java but also git, bash, wget etc

all easily installed on linux but a lot more complex on others thatswhy there is the "user installation" guide too :)

File: docs/core/deploy.md (hunk 1 above)
Comment (karussell) [inline docs/core/deploy.md]: makes sense

File: docs/web/quickstart.md (hunk 3 above)
Comment (boldtrn) [inline docs/web/quickstart.md]: Ok, didn't know that I will adapt it to your comment.

Comment (karussell) [inline docs/web/quickstart.md]: Sorry, what do you mean with "adapt it to your comment" :)

Comment (boldtrn) [inline docs/web/quickstart.md]: I will rephrase it to something like (from your first comment):

> "To see how GraphHopper is configured for production usage, see the separate site"

**Comment by boldtrn** [comment]
Ok, changed the docs, according to your comments. Hope its fine now.

**Comment by karussell** [comment]
Thanks!

**Comment by karussell** [comment]
What is this coveralls stuff - is this activated in your repo? Cannot find how to disable it.

**Comment by boldtrn** [comment]
> What is this coveralls stuff - is this activated in your repo? Cannot find how to disable it.

Do you mean me? No, the branch is deleted in my repo for 1.5 years now. I have no idea what happened here.

**Comment by karussell** [comment]
Wow, strange.
